Sources/Plasma/FeatureLib/pfPython/cyPythonInterface.cpp

```diff
@@ -725,7 +732,20 @@ class pyErrorRedirector
 
         // Send to the log server
         ST::utf16_buffer wdata = fData.to_string().to_utf16();
-        NetCliAuthLogPythonTraceback(wdata.data());
+        if (wdata.size() < kMaxTracebackLength - 1) {
+            NetCliAuthLogPythonTraceback(wdata.data());
+        } else {
+            // The traceback is too long for a single LogPythonTraceback message,
+            // so split it up into multiple small messages
+            // and add a truncation marker at the end of every message but the last.
+            auto pos = wdata.begin();
+            for (; wdata.end() - pos >= kMaxTracebackLength - 1; pos += kTracebackChunkLength - 1) {
+                ST::utf16_buffer chunk = ST::utf16_buffer(pos, kMaxTracebackLength - 1);
+                memcpy(&chunk[kTracebackChunkLength - 1], kTracebackTruncatedMarker, sizeof(kTracebackTruncatedMarker));
+                NetCliAuthLogPythonTraceback(chunk.data());
+            }
+            NetCliAuthLogPythonTraceback(pos);
+        }
```

Comment:
I wonder if it would be better to move this into `NetCliAuthLogPythonTraceback()`? I feel like users of this API should probably not be burdened with needing to chunk the traceback, even though this is currently the only user of the API.

Reply:
It would definitely be more convenient in case anything else wants to send Python errors. Right now we have this pattern that every `NetCli` function is a thin wrapper around a network message, so I didn't want to make one of them send multiple messages. But perhaps that doesn't matter much.

Reply:
There's now a new `NetComm` function that handles the chunk splitting before calling the `NetCli` function.